packages/sui-changelog/package.json

```diff
@@ -32,16 +32,11 @@
   },
   "devDependencies": {
     "@s-ui/lint": "3",
-    "@s-ui/mono": "1",
+    "@s-ui/mono": "1.43.0",
```

Comment:
really?

Reply:
That's very weird. :O I didn't add this line at all.

Reply:
Fixed ✔️